server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -536,6 +537,8 @@ public void endRound(final List<Elector> the_signatories) {
     if (my_current_round_index == null) {
       throw new IllegalStateException("no round to end");
     } else {
+      this.setAuditBoardCount(null);
```

Comment:
I guess `null` means we haven't picked a board size yet?

Reply:
Right, the client knows about this and won't let audit boards sign in until there's some value here. Clearing it out when the round ends should suffice. I'm going to have to test more thoroughly to see if there are any edge cases - we're doing client work today and tomorrow.